ddapm_test_agent/agent.py

```diff
@@ -459,6 +514,82 @@ async def handle_v2_apmtelemetry(self, request: Request) -> web.Response:
         # TODO: Snapshots
         return web.HTTPOk()
 
+    async def handle_put_tested_integrations(self, request: Request) -> web.Response:
+        # we need to store the request manually since this is not a real DD agent endpoint
+        await self._store_request(request)
+        await self.update_seen_integration_versions()
```

Comment:
An alternate solution for this would be to add an endpoint that dumps all the seen integrations to a specified file. That way the path can be customized if need-be.

This way we aren't writing to disk on every single request.